### ISSUE TITLE
Is it possible to pass the username and password without login page?

### ISSUE THREAD

**Issue by mut9bu** [prasathmani/tinyfilemanager#802]
Hello
First of all thank you so much for this application. It is one of the most efficient shared applications on the web i have seen.
We want to use the tinyfilemanager in our intranet. The clients are logged in to our intranet web pages with their domain acounts. (Windows authentication).  We want that they enter the tinyfilemanager without entering additional username and password.
In order to use multiple clients the tinyfilemanager  at the same time, i think we have to use the intern users definition of tinyfilemanager.
The only way at the moment is to give the username and password over the login page.
Is it not possible to pass the at least the username information without the login page? I think using url parameters is also not possible because they are resetted by tyinyfilemanager. Can you suggest me a workaround? Is something like auto login possible?

**Comment by prasathmani** [comment]
check this, it may help you https://www.youtube.com/watch?v=KEX2xenQENc

**Comment by mut9bu** [comment]
I have tried this. But for that I have to disable the authentication in Tinifilemanager ($Auth=false). How can I manage in this case a multi-user operation? I mean every user should be able to connect to different root path at the same time?

**Comment by mrlawrencelam** [comment]
Yes, I also hope I can skip the login for public users but still with auth enabled for the administrator to manage files.

I tried upload.php?username=user&password=user123 but it didn't work.
